res/layout/preview_audio_fragment.xml

```diff
@@ -62,4 +54,17 @@
 		android:layout_margin="16dp"
    	    />
 
+    <ProgressBar
+        android:id="@+id/transferProgressBar"
+        android:layout_width="match_parent"
+        android:layout_height="4dp"
+        android:padding="0dp"
+        android:layout_margin="0dp"
+        android:layout_alignParentTop="true"
+        style="@style/Widget.ownCloud.TopProgressBar"
+        android:indeterminate="false"
+        android:indeterminateOnly="false"
+        android:visibility="visible"
+        />
```

Comment:
This ```transferProgressBar``` is used in different layouts with the exact same attributes/values. So it could be put into its own layout definition file and just be included via a merge-Tag.